### DISCUSSION TITLE
Patternfly datatable column management with draggable example not working. I Want to use it in my project

### DISCUSSION THREAD

**Discussion by kamleshdarteweb** [patternfly/patternfly#5332] [Q&A] [answered]
Here is example link
https://www.patternfly.org/v4/components/table/react-demos/column-management-with-draggable

**Comment by mcoker** [comment, marked as answer]
Hey @kamleshdarteweb, thanks for letting us know! I've filed an issue in our [patternfly-react repo](https://github.com/patternfly/patternfly-react) here - https://github.com/patternfly/patternfly-react/issues/8547
